lib/ws_servers/api/handlers/on_remove_strategy.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+const PI = require('p-iteration')
+const send = require('../../../util/ws/send')
+const sendError = require('../../../util/ws/send_error')
+const { notifyInternalError, notifySuccess } = require('../../../util/ws/notify')
+const validateParams = require('../../../util/ws/validate_params')
+const isAuthorized = require('../../../util/ws/is_authorized')
+const capture = require('../../../capture')
+
+module.exports = async (server, ws, msg) => {
+  const { d, db } = server
+  const [, authToken, id] = msg
+  const validRequest = validateParams(ws, {
+    authToken: { type: 'string', v: authToken },
+    id: { type: 'string', v: id }
+  })
+
+  if (!validRequest) {
+    return
+  }
+
+  if (!isAuthorized(ws, authToken)) {
+    return sendError(ws, 'Unauthorized')
+  }
+
+  const { Strategy } = db
+  let strategiesByID
+
+  try {
+    strategiesByID = await Strategy.getAll()
```

Comment:
`await Strategy.rmById()` would be useful to have

same for `Strategy.getById()`

Reply:
just checked, receiving by id:

```
const strategy = await Strategy.find(['id', '=', id])
```

Reply:
same for `rm`?

Reply:
`rm` should be possible, too: https://github.com/bitfinexcom/bfx-hf-models-adapter-lowdb/blob/master/lib/map_methods/rm.js